ipn/ipnlocal/local.go

```diff
@@ -335,6 +335,9 @@ type LocalBackend struct {
 	// lastSuggestedExitNode stores the last suggested exit node ID and name.
 	// lastSuggestedExitNode updates whenever the suggestion changes.
 	lastSuggestedExitNode lastSuggestedExitNode
+
+	offlineAutoUpdateMu     sync.Mutex
```

Comment:
we try to limit the number of mutexes in a type, at least without lock precedence documentation.